test/utilities.go

```diff
@@ -0,0 +1,196 @@
+package test
+
+import (
+	"bufio"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/nsf/jsondiff"
+)
+
+// EqualJson compares two json strings.
+// returns true if considered equal, false otherwise.
+// The error returns the difference.
+// For reference: j1 is the baseline, j2 is the test
+func EqualJson(j1, j2 string) (ans bool, err error) {
```

Comment:
The file should already be sorted, does this test pass if you only sort the actual object? (j2)

Reply:
I did not check. But it is irrelevant. The sorted order is not valid if there are default value fields which are not in the actual object. The actual order sorted will have a different order.